doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -20,33 +20,35 @@ You need to have _cygwin_ installed and configured if you are running {PRODUCT}
 . After successful installation, add the path to _cygwin_ _bin_ folder to your environment PATH variable. E.g. *C:\cygwin64\bin*
 
 .Procedure
-
-. Log in to {CENTRAL}.
-. Select *Menu* -> *Projects*.
+. In {CENTRAL}, go to *Menu* -> *Projects*.
 . Select or create the space into which you want to import the projects.
 . Click image:project-data/dots.png[] on the right side of the screen and select *Import Project*.
-. In the *Import Project* window, enter the URL (for example, `\https://github.com/USERNAME/REPOSITORY_NAME.git`) and credentials for the Git repository that contains the project that you want to import and click *Import*.
+. In the *Import Project* window, enter the URL (for example, `\https://github.com/USERNAME/REPOSITORY_NAME.git`) and credentials for the GIT repository that contains the project that you want to import and click *Import*.
```

Comment:
No, it's indeed Git, as it was. If we want to go GIT, we'd have to change it globally. But Git is most common.

Reply:
Yes, my mistake, I picked up GIT from somewhere. Reverted these changes.